panel/src/main/scala/hmda/panel/PanelCsvLoader.scala

```diff
@@ -0,0 +1,101 @@
+package hmda.panel
+
+import java.io.File
+
+import akka.{ Done, NotUsed }
+import akka.actor.ActorSystem
+import akka.util.{ ByteString, Timeout }
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{ FileIO, Flow, Framing, Sink, Source }
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{ HttpEntity, _ }
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import com.typesafe.config.ConfigFactory
+import hmda.apiModel.protocol.admin.WriteInstitutionProtocol
+
+import scala.concurrent.duration._
+import hmda.parser.fi.InstitutionParser
+import org.slf4j.LoggerFactory
+import spray.json._
+
+import scala.util.Try
+
+object PanelCsvLoader extends WriteInstitutionProtocol {
+  implicit val system: ActorSystem = ActorSystem("hmda")
+  implicit val materializer = ActorMaterializer()
+  implicit val timeout: Timeout = Timeout(5.second)
+  implicit val ec = system.dispatcher
+  val log = LoggerFactory.getLogger("hmda")
+
+  val config = ConfigFactory.load()
+  val host = config.getString("hmda.adminHost")
+  val port = config.getInt("hmda.adminPort")
+
+  def main(args: Array[String]): Unit = {
+
+    if (args.length < 1) {
+      exitSys("No file argument provided", 1)
+    }
+
+    val file = new File(args(0))
+    if (!file.exists() || !file.isFile) {
+      exitSys("File does not exist", 2)
```

Comment:
I'm not entirely sure why the log doesn't print, but your logs do show the program exiting with the correct code (`2`).